code/responsive design/style.css

```diff
@@ -0,0 +1,179 @@
+body {
+    margin: 0;
+}
+
+#profile {
+    background-color: #f9b8b6;
+    padding: 1em;
+    margin: 1em;
+    margin-bottom: 2em;
+}
+
+#profile > img {
+    outline: 0.3rem solid #fff;
+    margin: 0.5em 0;
+    width: 100%;
+    max-width: 600px;
+    height: auto;
+    margin: 1rem 0;
+}
+
+#profile > span {
+    font-size: 1.3em;
+    font-weight: 600;
+    padding-bottom: 1em;
+}
+
+.card {
+    border: 3px solid #006d9e;
+    margin: 0.2rem;
+    height: auto;
+}
+
+img {
+    width: 100%;
+    height: auto;
+}
+
+header {
+    background-color: #a7cddf;
+    font-size: 2em;
+    display: flex;
+    justify-content: center;
+}
+ #page-wrapper {
+    padding: 1rem;
+ }
+
+ @media (min-width: 500px) {  
+    #page-wrapper {  
+        display: flex;  
+        flex-wrap: wrap;  
+        justify-content: center;  
+    }  
+
+    #profile {  
+        flex: 1 1 300px; 
+        margin: 1em;  
+    }  
+
+    #gallery {  
+        display: flex;  
+        flex-wrap: wrap;  
+      flex-direction: column;
+        flex: 3 1 500px; 
+    }  
+
+    .card {  
+        flex: 1 1 200px;  
+        margin: 0.5em; 
+    }
+    .card {  
```

Comment:
Keep an eye out for repeat selectors like this, all of these properties could be set within the single selector. Also, for example, you're setting a margin for .card and then immediately overwriting it with a different margin for .card.